src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+import List, { editTask, cleanAll } from './crud.js';
+import state from './interactive.js';
+
+const add = document.getElementById('send');
+const inputAdd = document.getElementById('add');
+const form = document.getElementById('form');
+const cleanBtn = document.getElementById('clean');
+
+const task = JSON.parse(localStorage.getItem('tasks')) || [];
```

Comment:
OPTIONAL
DRY (Don't Repeat Yourself) principle:
The task constant is defined twice in the code, which goes against the DRY principle. Instead, the code should define it once at the top and then use it throughout the code and the output doesn't change